node_modules/oae-core/groupprofile/css/groupprofile.css

```diff
@@ -0,0 +1,51 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+.groupprofile-widget .groupprofile-description {
+    white-space: pre-wrap;
+}
+
+.groupprofile-widget .groupprofile-description,
+.groupprofile-widget .groupprofile-description-empty {
+    margin-bottom: 35px;
+}
+
+.groupprofile-widget .groupprofile-description-empty button {
+    color: #888;
+    font-style: italic;
+    padding: 0 0 0 1px;
+    text-decoration: underline;
+}
+
+.groupprofile-widget .groupprofile-members {
+    margin-bottom: 15px;
+    margin-left: 4px;
+    width: 180px;
```

Comment:
Sure, but just to be clear, if you have 9 members showing, on desktop resolution you have one little lingering one (they go 4 wide)